tests/unit/rpc/handlers/BookChangesTests.cpp

```diff
@@ -222,3 +223,51 @@ TEST_F(RPCBookChangesHandlerTest, NormalPath)
         EXPECT_EQ(*output.result, json::parse(kEXPECTED_OUT));
     });
 }
+
+TEST_F(RPCBookChangesHandlerTest, NormalPathWithDomain)
+{
+    static constexpr auto kEXPECTED_OUT =
+        R"JSON({
```

Comment:
nit: Formatting of JSON could be improved a bit :)

Reply:
Improved in the whole file